tee-worker/cli/src/trusted_base_cli/commands/litentry/request_vc.rs

```diff
@@ -75,6 +75,9 @@ pub struct RequestVcCommand {
 	/// subcommand to define the vc type requested
 	#[clap(subcommand)]
 	command: Command,
+	/// mode for the request-vc
+	#[clap(short = 'i', long, default_value_t = false)]
+	isolated: bool,
```

Comment:
Shall we consider using `isolated` by default? Because this should become the standard way of requesting VCs

Maybe rename it to something opposite

Reply:
Renamed it to `stf` ✅